### ISSUE TITLE
Author full is not showing the full name

### ISSUE THREAD

**Issue by fabioperrella** [jonas/tig#560]
Given a git log like this:
```
git log --pretty=format:'%cN|%ce|%h'

Fabio Perrella|fabio.perrella@locaweb.com.br|ca737a2
Fabio Perrella|fabio.perrella@locaweb.com.br|fb097c5
Fabio Perrella|fabio.perrella@locaweb.com.br|29040b6
Fabio Perrella|fabio.perrella@locaweb.com.br|2b547cd
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|e11e5ed
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|2669aed
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|0571530
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|be75245
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|dc1f45e
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|e990fb5
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|b4b74b2
Fabio Perrella e Gustavo Santana|fabio.perrella@locaweb.com.br|b7e35a0
Fabio Perrella|fabio.perrella@locaweb.com.br|ab883a6
```

Tig is not showing the "full name" when is `Fabio Perrella e Gustavo Santana` (I use to change the user name when paring)

![image](https://cloud.githubusercontent.com/assets/302303/21849118/4a8a3810-d7eb-11e6-8c1e-1afc82ddcb11.png)

I tried to change setting and use author `abbreviated`, `email` and `email-use`, but none of them shows the author like `%cN` of git log.

Is it possible to show the real full name?

**Comment by jonas** [comment]
If I recall correctly `%cN` is the committer name. Tig uses the author name `%aN`.

**Comment by fabioperrella** [comment]
I've just tested `%aN` and the result is the same as `%cN`, so I think Tig do it in another way

```
git log --pretty=format:'%aN|%cN'
Fabio Perrella|Fabio Perrella
Fabio Perrella|Fabio Perrella
Fabio Perrella e Gustavo Santana|Fabio Perrella e Gustavo Santana
Fabio Perrella e Gustavo Santana|Fabio Perrella e Gustavo Santana
Fabio Perrella e Gustavo Santana|Fabio Perrella e Gustavo Santana

```

**Comment by jonas** [comment]
I think this has been fixed in commit 8eae2c380146d2803de86296518d32851c37182c
That commit changes the author cache to use both the name and email as a key.

**Comment by jonas** [comment]
Would it be possible for you to test that out? I can roll a new version if that would be easier for you.

**Comment by fabioperrella** [comment]
Sure! how can I update?

I'm using the version from ubuntu repo:
```
apt-cache policy tig
tig:
  Installed: 2.0.2-2build1
  Candidate: 2.0.2-2build1
  Version table:
 *** 2.0.2-2build1 500
        500 http://br.archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by jonas** [comment]
The commit referenced above was released in version 2.2.2.
Please test that version and report any issues.
Closing this for now.
